Providers/nxOMSAutomationWorker/automationworker/worker/runtime.py

```diff
@@ -52,7 +52,10 @@ def start_runbook_subprocess(self):
                 if self.runbook.definition_kind_str == "PowerShell" and parameter["Name"]:
                     # Handle named parameters for PowerShell arriving out of order
                     cmd += ["-%s" % parameter["Name"]]
-                cmd += [str(json.loads(parameter["Value"]))]
+                try:
+                    cmd += [str(json.loads(parameter["Value"]))]
```

Comment:
Normally with python, it isn't advised to do a str(dict) as it is not same as serialization, did this use to work?

https://stackoverflow.com/questions/47648886/difference-between-strdict-and-json-dumpsdict